packages/msw-addon/package.json

```diff
@@ -40,7 +40,7 @@
     "typescript": "^4.5.2"
   },
   "peerDependencies": {
-    "msw": ">=0.35.0 <1.0.0"
+    "msw": ">=0.35.0 <1.1.0"
```

Comment:
@rajtslegr taking into account that `msw` follows [semver](https://semver.org/) we can safely make the peerDependencies as `<2.0.0`

`msw` semver statement: https://github.com/mswjs/msw/blob/main/CHANGELOG.md

Reply:
Thanks for suggestion! Done!